release/rancher/rancher.go

```diff
@@ -183,60 +202,97 @@ func rancherHelmChartVersions(repoURL string) ([]string, error) {
 }
 
 func SetKDMBranchReferences(ctx context.Context, forkPath, rancherBaseBranch, currentKDMBranch, newKDMBranch, forkOwner, githubToken string, createPR, dryRun bool) error {
-	if _, err := os.Stat(forkPath); err != nil {
-		return err
-	}
-	scriptPath := filepath.Join(forkPath, setKDMBranchReferencesScriptFile)
-	f, err := os.Create(scriptPath)
-	if err != nil {
-		return err
+	branchName := "kdm-set-" + newKDMBranch
```

Comment:
Can this be a const?

Reply:
No, because `newKDMBranch` is not a const